02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -79,22 +98,22 @@ For this example analysis, we will use this [CREB overexpression zebrafish exper
 
 ## Check out our file structure!
 
-Your new analysis folder should contain: 
+Your new analysis folder should contain:
 
 - The example analysis `.Rmd` you downloaded  
-- A folder called `data` (currently empty) 
+- A folder called `data` (currently empty)
 - A folder for `plots` (currently empty)  
 - A folder for `results` (currently empty)  
-    
+
 Your example analysis folder should contain your `.Rmd` and three empty folders (which won't be empty for long!).
 
-If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds). 
+If the concept of a "file path" is unfamiliar to you; we recommend taking a look at our [section about file paths](https://alexslemonade.github.io/refinebio-examples/01-getting-started/getting-started.html#an-important-note-about-file-paths-and-Rmds).
```

Comment:
Oh, I should have made it clear, no actual changes have been made here. Github just put it because of a weird spacing thing. 

This same sentence is in the template so is in many other places. I'm going to temporarily ignore this punctuation change since I'm not sure its worth the effort (for now) until we find more things we want changed in the template. But I'll keep in mind.

Reply:
Oh, I put a warning in my PR description, but it could be easily missed: 

> Note The Files Changed is showing weird spacing things, but there are no actual changes after line 47 so you don't need to review beyond that point.